Probesys/bileto
# new: Allow to assign a ticket to a team

## Related issue(s)



https://github.com/Probesys/bileto/issues/576

## How to test manually



1. create a team with an authorization on a specific organization and a different user than the current one as an agent
2. create a ticket in this organization and assign the ticket to the team (not on the user)
3. create another ticket in this organization, but do not assign the ticket (no team or user)
4. make sure the role of the authorization don't have the "See all the tickets" permission
5. login with the agent of the team
6. check that you can see the first ticket, but not the second

## Checklist
